docker/compose/docker-compose-wtih-existing-results-db.yml

```diff
@@ -0,0 +1,49 @@
+version: "2"
```

Comment:
please add comment to the beginning of the both .yml files:
- this one for the case, when ...
- this one ...